### DISCUSSION TITLE
Is there a  developer version of implemented changes list?

### DISCUSSION THREAD

**Discussion by locksoft** [diasurgical/DevilutionX#7031] [Q&A] [answered]
Sorry guys, another quick question: is there a page that shows what changes are planned for the next version (1.6.0) and witch one is already been added so far to the developer build, so to know what to check? I'm almost sure I've ended one in a page showing a series of checkboxes for 1.6.0 and maybe 1.6.1 (and a total percentage of stuff done) for planned changes, but I can't find it anymore. Or maybe I've just dreamed that?
I installed 1.6.0b and I can't really say any difference (map seems a bit different, but I'm not sure).
Thanks.

**Comment by AJenbo** [comment, marked as answer]
Check milestones for plans:
https://github.com/diasurgical/devilutionX/milestones

There might be additional items that where merged, these will eventually be listed at the top here:
https://github.com/diasurgical/devilutionX/blob/master/docs/CHANGELOG.md

1.6.1 would be bug fixes for mistakes made in 1.6.0 so we can't really plan that before we know what we did wrong.

**Reply by locksoft** [reply]
Thank you, that's exactly the page I was looking for. :)
